sdk/datafactory/Azure.ResourceManager.DataFactory/tests/Scenario/DataFactoryDatasetResourceTests.cs

```diff
@@ -3,6 +3,7 @@
 
 using System.Threading.Tasks;
 using Azure.Core;
+using Azure.Core.Expressions.DataFactory;
```

Comment:
This will be needed once https://github.com/Azure/autorest.csharp/pull/3553 lands to prevent regen issues, since the types being used in these tests will now be coming from this namespace.